code/modules/roguetown/roguejobs/alchemist/reagents.dm

```diff
@@ -3,7 +3,7 @@
 	description = "Gradually regenerates all types of damage."
 	reagent_state = LIQUID
 	color = "#ff0000"
-	taste_description = "red"
+	taste_description = "health"
```

Comment:
Good, but mention such minor changes on the sidelines

Reply:
👍